integration-tests/pom.xml

```diff
@@ -94,20 +94,27 @@
 					</execution>
 
 					<execution>
-						<id>install-sidecar</id>
+						<id>cds.build-sidecar</id>
 						<goals>
 							<goal>npm</goal>
 						</goals>
-						<phase>pre-integration-test</phase>
 						<configuration>
-							<arguments>install --no-save</arguments>
+							<arguments>run build</arguments>
 							<skip>${skipTests}</skip>
 							<workingDirectory>${sidecar.dir}</workingDirectory>
 						</configuration>
 					</execution>
 				</executions>
 			</plugin>
 
+			<!-- Introduces a hack: MTX sidecar is executed in the pseudo-production mode
```

Comment:
There must be a better way, I think. Did you check if setting 

```
"cds": {
  "cdsc": {
    "moduleLookupDirectories": [ "node_modules/", "target/cds/" ]
  }
}
```

in mtx/sidecar/package.json would help? Maybe also only for development profile (not sure though why it works in production, but not in development profile, smells like a bug in MTX Sidecar)